src/rest/api_call.cpp

```diff
@@ -43,7 +45,8 @@ void rest_client::perform_request(std::string uri, rest_request_type method, htt
         if(response.code == 429) {  // Being rate limited
             int wait_for = message["retry_after"].get<int>();
             debug::log(debug::log_level::INFORMATIONAL, debug::log_origin::REST, "API request exceeds rate limts. Retrying in " + std::to_string(wait_for) + "ms");
-            boost::this_thread::sleep(boost::posix_time::milliseconds(wait_for));
+            // This is kinda confusing but we have to specify when to wait.
```

Comment:
What do you mean by 'kinda confusing'?

Reply:
ok I removed the comment, originally I used a larger chrono function call and `std::this_thread::sleep_for` instead of `std::this_thread::sleep_until`